src/Components/NToastNotifyViewComponent.cs

```diff
@@ -18,7 +18,7 @@ public NToastNotifyViewComponent(IToastNotification toastNotification, ILibrary
             _nToastNotifyOption = nToastNotifyOption;
         }
 
-        public IViewComponentResult Invoke()
+        public IViewComponentResult Invoke(string? nonce = null)
```

Comment:
How is the value of this parameter `nonce` provided ?

Reply:
That can be provided in the client code like this:
`@await Component.InvokeAsync("NToastNotify", new { nonce = "2726c7f26c" })`